qiskit/synthesis/two_qubit/two_qubit_decompose.py

```diff
@@ -495,11 +498,10 @@ def specialize(self):
 
 
 class TwoQubitWeylSWAPEquiv(TwoQubitWeylDecomposition):
-    """U ~ Ud(𝜋/4, 𝜋/4, 𝜋/4) ~ U(𝜋/4, 𝜋/4, -𝜋/4) ~ SWAP
+    r""":math:`U \sim U_d(\pi/4, \pi/4, \pi/4) \sim U(\pi/4, \pi/4, -\pi/4) \sim \text{SWAP}`
 
     This gate binds 0 parameters, we make it canonical by setting
-        K2l = Id ,
-        K2r = Id .
+    :math:`K2l = Id` , :math:`K2_r = Id`.
```

Comment:
It's not API-documented so it doesn't matter so much, but here (and some other places) you have `K2l` but `K2_r`